method/include/boost.h

```diff
@@ -11,12 +11,11 @@ std::vector<std::string> getTsvFiles(const po::variables_map &vm);
 std::string              getBed(const po::variables_map &vm);
```

Comment:
worth adding to the documentation (and `--help` flag) how do the TSVs look like, e.g. which column is `methylated reads` and which column is `total number of reads`? Worth adding a quick check `total number of reads > methylated reads` to be on the safe side?

Reply:
added some more information to the help flag and the checking for the number of reads as well